modules/apps/transfer/module.go

```diff
@@ -160,6 +160,6 @@ func (am AppModule) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
 }
 
 // WeightedOperations returns the all the transfer module operations with their respective weights.
-func (am AppModule) WeightedOperations(_ module.SimulationState) []simtypes.WeightedOperation {
+func (AppModule) WeightedOperations(_ module.SimulationState) []simtypes.WeightedOperation {
```

Comment:
let me know if this is right.  I think that in the end the _'s are kinda an anti pattern, I've found myself searching for the right name.